Monika After Story/game/script-topics.rpy

```diff
@@ -7591,7 +7591,91 @@ label monika_cartravel:
     m 1hua "Just the idea of travelling around the world makes me so excited!"
     m 1eka "I really can't wait, [player]~"
     return
-
+    
+    ###poems archives
+    
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_old_poem",
+            category=['literature'],
+            prompt="Old poems",
+            random=True,
+            aff_range=(mas_aff.AFFECTIONATE,None)
```

Comment:
Not really sure why showing poems should be limited to affectionate+. Seems a little strange since you were shown all of these at base ddlc, i.e. where no affection level existed. I understand > Normal as a condition. But affectionate seems weird.

Reply:
All right, I change it for normal. I agree that Affectionate is too strong. Maybe use Happy rather than Normal? I use this function especially for the Valentine poems, Monika must believe in the player before showing these "treasures"

Reply:
I don't really agree, the valentines day poem was shown to you only if you were >= 0 affection.

D25 was enamored, but either way, you'd have to have unlocked these poems (seen them through their event) for you to see them again in the first place. So I don't see a problem with having it under normal.

Reply:
Ok, I change it to normal.